_docs_integrate/61-data-model.md

```diff
@@ -42,6 +42,7 @@ A token has the following properties:
 | expiresAt          | `string`  | {% include descr_expiresAt class="Token" %}                   |                                               |
 | secretKey          | `string`  | {% include descr_secretKey class="Token" %}                   | saved only locally                            |
 | truncatedReference | `string`  | {% include descr_truncatedReference class="Token" %}          | saved only locally                            |
+| isEphemeral        | `boolean` | //TODO                                                        |
```

Comment:
Mögliche Beschreibung: 

``If set to `true` the token will will not be cached in the database and only displayed once. You will not be able to fetch this Token unless you remember the id and secretKey of the Token.``